.github/ISSUE_TEMPLATE/feature-request.yml

```
@@ -0,0 +1,17 @@
+name: Feature request
+description: Check instructions for submitting your idea on the mailing list first.
+title: "FEATURE: <Please write a comprehensive title after the 'FEATURE: ' prefix>"
```

Those prefixes aren't needed here :)

```suggestion
title: "<Please write a comprehensive title>"
```